source/features/default-branch-button.tsx

```diff
@@ -51,6 +51,10 @@ async function init(): Promise<false | void> {
 	branchSelector.parentElement!.before(defaultLink);
 	groupButtons([defaultLink, branchSelector.parentElement!]).classList.add('d-flex');
 	branchSelector.style.float = 'none'; // Pre "Repository refresh" layout
+
+	// Fix #4240
+	branchSelector.style.zIndex = 'auto';
+	branchSelector.parentElement!.style.zIndex = 'auto';
```

Comment:
The first line doesn't seem necessary. Also this order makes aligns the two `branchSelector.parentElement!.` calls

```suggestion
	branchSelector.parentElement!.style.zIndex = 'auto'; // For #4240
	groupButtons([defaultLink, branchSelector.parentElement!]).classList.add('d-flex');
	branchSelector.style.float = 'none'; // Pre "Repository refresh" layout
```